styles.css

```diff
@@ -0,0 +1,2153 @@
+    gap: 12px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    padding: 0;
+    position: absolute;
+    width: 26%;
+    height: 40px;
+    right: 8px;
+    top: -20px;
+  }
+
+  .navbar-brand {
+    width: 20%;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  #headline {
+    background: url('images/Header.png');
+    background-size: 100% 100%;
+    background-position: 50px;
+    background-repeat: no-repeat;
+    background-color: white;
+    top: 12.5%;
+  }
+
+  .logo-placeholder {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    width: 100px;
+    height: 40px;
+    left: 143px;
+    top: -20px;
+  }
+
+  #break {
+    display: block;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 2px;
+    position: absolute;
+    width: 100%;
+    height: 268px;
+    left: 260px;
+    top: calc(40% - 268px / 2);
+  }
+
+  .primary-text1 {
+    width: 100%;
+    height: 122px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .primary-text2 {
+    width: 58%;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .socialmedia {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 16px;
+    width: 30%;
+    height: 24px;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .socialmedia-icons {
+    position: absolute;
+    width: 170.17px;
+    height: 20px;
+    left: 260px;
+    top: 430px;
+  }
+
+  .twitter {
+    position: absolute;
+    left: 1.06%;
+    right: 80.56%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .linkedin {
+    position: absolute;
+    left: 18.89%;
+    right: 70.13%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .m {
+    position: absolute;
+    left: 40.54%;
+    right: 75.07%;
+    top: 20.57%;
+    bottom: 86.12%;
+  }
+
+  .github {
+    position: absolute;
+    left: 63.32%;
+    right: 72.43%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .vector {
+    position: absolute;
+    left: 85.89%;
+    right: 70.13%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .grid-container {
```

Comment:
- [ ] Please ensure that the space between the cards are respected as shown in the Figma Design.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221351501-d6360a9f-892f-4e8f-9adb-1a3bcfadabf5.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221351533-6418e426-3713-4e2f-96e1-5ac05689a681.png)